packages/grafana-ui/src/components/Portal/Portal.tsx

```diff
@@ -1,5 +1,6 @@
 ﻿import React, { PureComponent } from 'react';
 import ReactDOM from 'react-dom';
+import { appZIndexes } from '../../themes/default';
```

Comment:
Feels cleaner to rewrite this a a functional component and use useTheme2 to access the theme object (that has zIndex)